spdk.spec

```
@@ -20,7 +20,7 @@ Source:		https://github.com/%{name}/%{name}/archive/v%{version}.tar.gz
 
 Patch0:		0001-env_dpdk-tokenize-env_context.patch
 Patch2:		0002-vmd-update-for-changes-in-IceLake-platform.patch
-Patch3:                 0001-blob-use-uint64_t-for-unmap-and-write_zeroes-lba-cou.patch
+Patch3:         0003-blob-use-uint64_t-for-unmap-and-write_zeroes-lba-cou.patch
```

Still seems to be some kind of whitespace issue here.  Looks like the previous lines are using tabs and you are using spaces.  You probably have tab expansion set at something other than 8.  I didn't realize that tabs were being used and probably messed you up in my previous suggestion.  This one should fix it I think:

```suggestion
Patch3:		0003-blob-use-uint64_t-for-unmap-and-write_zeroes-lba-cou.patch
```